Mixpanel/Mixpanel.m

```diff
@@ -741,6 +813,36 @@ - (void)dispatchOnNetworkQueue:(void (^)(void))dispatchBlock {
     });
 }
 
+- (void)optOutTracking
```

Comment:
We are missing a couple of things here: we need to call `deleteUser` and `clearCharges` in the `people` instance :)